README.md

```diff
@@ -80,6 +80,9 @@ docker-compose --project dockerComposeProjectName --file docker-compose.yml --pr
 
 `docker.service` is the exact name of the service as it appears in docker-compos.yml
 
+The default behaviour is to use `docker-compose exec` to invoke a command on a running container.
+You can change this to use `docker-compose run` for containers which by default are not running, with the property `composer.command`.
```

Comment:
"composer.command" should be "compose.command"

Reply:
Thanks for the spot – fixed (and also the typo in the line above, `docker-compos.yml`).